src/Swashbuckle.AspNetCore.SwaggerGen/XmlComments/XPathNavigatorExtensions.cs

```diff
@@ -0,0 +1,22 @@
+using System.Linq;
+using System.Xml.XPath;
+
+namespace Swashbuckle.AspNetCore.SwaggerGen
+{
+    internal static class XPathNavigatorExtensions
+    {
+        internal static XPathNavigator SelectFirstChild(this XPathNavigator navigator, string name)
+        {
+            return navigator.SelectChildren(name, "")
+                    ?.Cast<XPathNavigator>()
```

Comment:
Are the children here on line 18 guaranteed to always be an `XPathNavigator`?

Reply:
Can you code it in a way that doesn't assume internal implementation details? The runtime could theoretically change the implementation - would `OfType<XPathNavigator>()` do the same but in a way that won't throw with an `InvalidCastException` is that happened?

Reply:
I changed it  to OfType.